src/js/tabs/cli.js

```diff
@@ -193,6 +193,14 @@ tab.initialize = function (callback) {
             }
         });
 
+        $('.tab-cli .exit').click(function() {
+            TABS.cli.send("exit\n", function () { });
```

Comment:
You cannot just use `TABS.cli.send` -- this requires the accounting for the `cliBuffer`, as the FC might have a partial command "memorized" which would lead to bad behavior.